### PR TITLE
Build /ai/mlflow

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Built new page as per [doc](https://docs.google.com/document/d/1kafEsmi8OpaCD-WK4XnIBMgMStHUJNhM86nBN91e4ZI/edit) and [mock-up](https://www.figma.com/file/sGeCjFvA1kV9Kmvg2S2VBn/23.10-U.com---AI-bubble?node-id=1338%3A6595&mode=dev)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13149.demos.haus/ai/mlflow
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-6077

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
why this margin? it should be 0 
image

**Comment by mtruj013** [comment]
> why this margin? it should be 0

Because it seemed kind of misaligned with the suru without it. Anyway I've removed it

**Comment by mtruj013** [comment]
@lyubomir-popov @juanruitina ready for y'all but a quick note about this section:
image
 it originally asked for an image that didn't exist, the version that was on the copy doc was made by the pdm for a slide deck. @camillet16 was able to quickly spin up a better version. The pdm has approved the image and I've moved it the top of the col per Camille's suggestion, please let me know if you're happy with this

**Comment by juanruitina** [comment]
- [ ] Can you please update the navigation for the bubble to match the one in the design? So it makes more sense.

Screenshot 2023-09-15 at 10 45 09

- [ ] Please turn the H2-like CTA links into `p` elements.
- [ ] Add some margin below the MLflow logo so it doesn't look too tight on mobile, shallow one would suffice.

Finally, this is visual, but I understand given @lyubomir-popov's direction that we should drop the white background from the two first CTA sections and add an HR above. The last one should remain as it is.

**Comment by lyubomir-popov** [comment]
we don't desaturate logos, ideally these should be in full colour:
image
please do this as we've agreed, lines as on the ubuntu pro mockup and no white:
image
same here 
image

The lkast one is fine as it is at the bottom.

**Comment by mtruj013** [comment]
@juanruitina could you take another look?

**Review by albertkol — APPROVED** [review]
:+1: Well done!

File: templates/ai/mlflow.html [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+  <div class="row--50-50">
+    <hr class="p-rule">
+    <div class="col">
+      <div class="p-section--shallow">
+        <h2>Deploy easily on any infra, from workstations to public clouds</h2>
+      </div>
+    </div>
+    <div class="col">
+      <div class="p-strip is-shallow">
+        <img src="https://assets.ubuntu.com/v1/b21b2f8a-ai-mlflow.svg">
```
Comment (anthonydillon) [inline templates/ai/mlflow.html:185]: This image is missing an alt

File: templates/ai/mlflow.html [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+
+<section class="p-strip--white is-deep">
+  <div class="u-fixed-width">
+    <p class="p-heading--2">Start your machine learning journey with open source</p>
+    <a class="p-button--positive" href="https://documentation.ubuntu.com/charmed-mlflow">Install Charmed MLflow</a>
+    <a class="p-button js-invoke-modal" href="/ai/contact-us">Contact us</a>
+  </div>
+</section>
+<!-- Set default Marketo information for contact form below-->
+<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/ai" data-form-id="3231" data-lp-id="6279" data-return-url="https://ubuntu.com/ai/thank-you?product=ai-managed" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
```
Comment (anthonydillon) [inline templates/ai/mlflow.html:316]: This one is missing the closing div.

**Review by anthonydillon — APPROVED** [review]
LGTM with a couple of comments

**Pull request merged**